src/Model/Response/Metadata/EmptyMetadataTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Model\Response\Metadata;
+
+trait EmptyMetadataTrait
```

Comment:
to me it would be more consistent to be also `MetadataAwareTrait` - sounds a bit like metadata would be empty when using this one, but in reality it's just initializing that

Reply:
Thanks! You are right, i rename it 👍🏼